cookbook/bundles/configuration.rst

```diff
@@ -294,8 +294,8 @@ The ``config:dump-reference`` command dumps the default configuration of a
 bundle in the console using the Yaml format.
 
 As long as your bundle's configuration is located in the standard location
-(``YourBundle\DependencyInjection\Configuration``) and does not require
-arguments to be passed to the constructor it will work automatically. If you
+(``YourBundle\DependencyInjection\Configuration``) and does not extend
+its constructor it will work automatically. If you
```

Comment:
"extend" sounds strange when we talk about methods imo. What about saying something like "and does have a constructor"?